xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -453,8 +453,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the maxiumum response header length.</summary>
+        <value>The maximum response header length.</value>
```

Comment:
In bytes, I assume?

Reply:
Actually, no.  The values are in kilobytes.

It is similar to this property:
https://msdn.microsoft.com/en-us/library/system.net.http.webrequesthandler.maxresponseheaderslength(v=vs.110).aspx

Reply:
Great, thanks! Since it's an integer, is the value rounded up or down? I would think it matters, but maybe it doesn't.

Reply:
The value is not rounded up or down.  The value specifies the number of kilobytes allowed for the maximum response headers length.

If "64" is set for this property, then 65536 bytes are allowed for the maximum response headers length.

Reply:
I see, makes sense! :)